tests/integration/test_mysql_database_engine/test.py

```diff
@@ -167,6 +167,27 @@ def test_bad_arguments_for_mysql_database_engine(started_cluster):
         assert 'Database engine MySQL requested literal argument.' in str(exception.value)
         mysql_node.query("DROP DATABASE test_bad_arguments")
 
+def test_column_comments_for_mysql_database_engine(started_cluster):
+    with contextlib.closing(MySQLNodeInstance('root', 'clickhouse', started_cluster.mysql_ip, started_cluster.mysql_port)) as mysql_node:
+        mysql_node.query("DROP DATABASE IF EXISTS test_database")
+        mysql_node.query("CREATE DATABASE test_database DEFAULT CHARACTER SET 'utf8'")
+
+        clickhouse_node.query(
+            "CREATE DATABASE test_database ENGINE = MySQL('mysql57:3306', 'test_database', 'root', 'clickhouse')")
```

Comment:
>  Flaky tests found: fail: 1

@sand6255 Need to drop this database at the end of test, since in CI all tests are run un random order => it broke another test.